app/com/lynxanalytics/biggraph/frontend_operations/AttributePropagationOperations.scala

```diff
@@ -41,11 +41,11 @@ class AttributePropagationOperations(env: SparkFreeEnvironment) extends ProjectO
       def apply() = {
         val weightName = params("weight")
         val weight = parent.vertexAttributes(weightName).runtimeSafeCast[Double]
-        for ((attr, choice) <- parseAggregateParams(params)) {
+        for ((attr, choice, name) <- parseAggregateParams(params, weight = weightName)) {
```

Comment:
Looking at this code it occurred to me that it seems inconsistent that we don't allow specifying a prefix in case of the aggregations towards segmentations. Not that I ever really missed it in practice, but maybe we should just add a prefix parameter in these two operations as well, defaulting to empty string?

Reply:
If you do this, you could avoid the prefix parameter for `parseAggregateParams`. Instead, it can take it from params. And the parameter could be added by `aggregateParams`. It would take a new argument with the default value for prefix, defaulting to `""`.

But all this is not too important, if there is some complication I don't see then it's fine to ignore this.

Reply:
> If you do this, you could avoid the prefix parameter for `parseAggregateParams`. Instead, it can take it from params. And the parameter could be added by `aggregateParams`. It would take a new argument with the default value for prefix, defaulting to `""`.

Very nice, thanks! Done.